lambdas/issue-credential/src/time-handler.ts

```diff
@@ -4,48 +4,56 @@ import { TimeEvent } from "./time-event";
 
 const logger = new Logger();
 
-const Unit = {
-  Seconds: "seconds",
-  Minutes: "minutes",
-  Hours: "hours",
-  Days: "days",
-  Months: "months",
-  Years: "years",
-  None: "none",
-};
+enum Unit {
+  Seconds = "seconds",
+  Minutes = "minutes",
+  Hours = "hours",
+  Days = "days",
+  Months = "months",
+  Years = "years",
+  None = "none",
+}
+
+export class TimeHandler implements LambdaInterface {
+  private readonly timeInSecs: number;
 
-const convert = (unit?: string): number => {
-  switch (unit) {
-    case Unit.Seconds:
-      return 1000;
-    case Unit.Minutes:
-      return 1000 * 60;
-    case Unit.Hours:
-      return 1000 * 60 * 60;
-    case Unit.Days:
-      return 1000 * 60 * 60 * 24;
-    case Unit.Months:
-      return 1000 * 60 * 60 * 24 * 30;
-    case Unit.Years:
-      return 1000 * 60 * 60 * 24 * 365;
-    case Unit.None:
-      return 1;
-    default:
-      throw new Error(`Unexpected time-to-live unit encountered: ${unit}`);
+  constructor() {
+    this.timeInSecs = this.msToSeconds(Date.now());
```

Comment:
Shouldn't this be captured in the handler?
If the lambda persists or there's a delay between instantiation and hadling a request, then subsequent requets would get an old value. Since this uses `Date.now`, it should be as recent as possible.